www/_static/swift.css

```diff
@@ -1,64 +1,104 @@
 /* Common styles */
-.body { color: black; background: white; margin: 0 0 0 0 }
+.body {
+  color: black;
+  background: white;
+  margin: 0 0 0 0;
+}
 
 /* No borders on image links */
-a:link img, a:visited img { border-style: none }
+a:link img,
+a:visited img {
+  border-style: none;
+}
 
-address img { float: right; width: 88px; height: 31px; }
-address     { clear: right; }
+address img {
+  float: right;
+  width: 88px;
+  height: 31px;
+}
 
+address {
+  clear: right;
+}
+
+table {
+  border: 2px solid black;
+  border-collapse: collapse;
+  margin-top: 1em;
+  margin-left: 1em;
+  margin-right: 1em;
+  margin-bottom: 1em;
```

Comment:
I prefer being explicit. I prefer this notation over short hand so didn't want to change it.